### ISSUE TITLE
Not usable

### ISSUE THREAD

**Issue by rex2630** [Evgenius33/pillbox-beds#1]
I changed control for lying to E and when i press E anywhere it will teleport me out of textures or when im really far from bed then it will teleport me there....

**Comment by BUTMAN133** [comment]
Me too, need to fix

**Comment by MatinDark** [comment]
Do you guys find any fix for that?

**Comment by rex2630** [comment]
No. I just made a quick fix by deleting the script :D . I wil rather use coords to do that...
